src/Autosuggest.js

```diff
@@ -235,15 +252,19 @@ class Autosuggest extends Component {
           case 'Enter': {
             const focusedSuggestion = this.getFocusedSuggestion();
 
-            if (focusedSuggestion !== null) {
+            if (focusedSuggestion) {
```

Comment:
This is intentionally changed because it's possible for `focusedSuggestion` to be `undefined` instead of `null`. We could change it to: 

`focusedSuggestion !== null && typeof(focusedSuggestion) !== 'undefined'`

but since this is always an object reference there's no gain there over the implicit check. Totally your call on the semantics but we do need to handle the `undefined` case here.